src/app/assembly-bench/bench-detail/bench-detail.component.html

```diff
@@ -24,6 +24,9 @@ <h1>{{pc.name}}</h1>
 </div>
 <div class="row">
   <div class="col-xs-12">
-    Parts
+    <p>{{pc.cpu}}</p>
```

Comment:
If `pc` is undefined then you gonna receive an error. What could you do as a solution?